save_sphere.py

```diff
@@ -274,7 +274,7 @@ def get_save_sphere_settings(actor_index: int):
                     results[map_num][diag_num][actor_num]["y"],
                     results[map_num][diag_num][actor_num]["diag"],
                 ]
-
+    print(ret_array)

```

Comment:
At the risk of starting another nuclear winter, this should specific `print` should either be removed or updated to use logging.